ARCHITECTURE.md

```diff
@@ -205,6 +207,19 @@ of bare pointers to data arrays, array sizes, etc... The ABI for these calls is
 driver functions may be templated on functor classes and/or accept lightwight C++ objects as
 parameters (such as `BoxDim`).
 
+## Autotuning
+
+HOOMD-blue automatically tunes kernel block sizes, threads per particle, and other kernel launch
+paramters. The `Autotuner` class manages the sparse multi-dimensional of parameters for each kernel.
+It dynamically cycles through the possible parameters and records the performance of each using CUDA
+events. After scanning through all parameters, it selects the best performing one to continue
+executing. GPU code in HOOMD-blue should instantiate and use one `Autotuner` for each kernel.
+Classes that use the autotuner should inherit from `Autotuned` which tracks all the autotuners and
```

Comment:
How does one know which class is autotuned?

Reply:
When a class has `Autotuner` member variables. I reworded the documentation to indicate this.